frontend/src/components/App/Home/RecentClusters.tsx

```diff
@@ -8,6 +9,13 @@ import { createRouteURL } from '../../../lib/router';
 import { getClusterPrefixedPath } from '../../../lib/util';
 import SquareButton from './SquareButton';
 
+const ToggleButton = styled(MuiToggledButton)({
+  textTransform: 'none',
+});
+
+/** Allow selecting multiple recent clusters. */
+const MULTI_ENABLED = import.meta.env.REACT_APP_MULTI_ENABLED === 'true' || true;
```

Comment:
I am willing to merge this PR as a temporary way to test this, but seems like this is enabled all the time.